src/relations.jl

```diff
@@ -1015,6 +1033,27 @@ function q_vap_saturation_from_pressure(
     return p_v_sat / (ρ * _R_v * T)
 end
 
+"""
+    q_vap_saturation_from_partial_pressures(param_set, q_tot, p, p_v_sat)
```

Comment:
Let's update this doc string to match the code.

Reply:
Should we mention something about it being added for the purpose of being compatible with anelastic approximation here?

Reply:
Let's add temperature to the argument list

Reply:
And replace `p_v_sat` with `phase`